examples/sample-input-byo.tfvars

```diff
@@ -0,0 +1,117 @@
+# !NOTE! - These are only a subset of variables.tf provided for sample.
+# Customize this file to add any variables from 'variables.tf' that you want 
+# to change their default values. 
+
+# ****************  REQUIRED VARIABLES  ****************
+# These required variables' values MUST be provided by the User
+prefix                                  = "<prefix-value>"
+location                                = "<aws-location-value>" # e.g., "us-east-1"
+ssh_public_key                          = "~/.ssh/id_rsa.pub"
+# ****************  REQUIRED VARIABLES  ****************
+
+# Bring your own existing resources
+vpc_id  = "<existing-vpc-id>" # only needed if using pre-existing VPC
+subnet_ids = {  # only needed if using pre-existing subnets
+  "public" : ["list-of-existing-public-subnets"],
+  "private" : ["list-of-existing-private-subnet-ids"],
+  "database" : ["list-of-existing-database-subnet-ids"]
+}
+security_group_id = "<existing-vpc-id>" # only needed if using pre-existing Security Group
```

Comment:
Shoud this be, security_group_id = "<existing-security-group-id>" ?

Reply:
Fixed